### DISCUSSION TITLE
[Prettier Plugin] Pug support

### DISCUSSION THREAD

**Discussion by pspeter3** [tailwindlabs/tailwindcss#7572] [Ideas]
Prettier supports Pug as an official plugin. It would be great if the Tailwind CSS sorter could also sort Pug classes.

**Comment by pspeter3** [comment]
I see it's on the TODO list https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/main/src/index.js#L32. Is the work to add it to the parsers list? https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/main/src/index.js#L349

**Comment by littleyellowspider** [comment]
I would like to see it as well. All my markup is currently generated dynamically using PUG. In conjunction with NodeJS and Tailwind CSS it provides a really elegant clean and importantly flexible technology stack.

**Comment by pspeter3** [comment]
It seems like the TODO comment for Pug support was removed in https://github.com/tailwindlabs/prettier-plugin-tailwindcss/commit/88310f37e2a8f12575ffe9db434c1b1cc1583d9a by @bradlc. MDX & Svelte are now available though.

**Comment by sandraroz** [comment]
Bump!
Would love to see it as well. We just converted our entire repository to using pug and we're obsessed with sorting things.
